service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsController.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import com.google.common.collect.ImmutableList;
+import io.swagger.annotations.Api;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBookingDetail;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.MapperService;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+@Api
+@RestController
+@RequestMapping("/callback/add-hearing-booking")
+public class HearingBookingDetailsController {
+    private final MapperService mapperService;
+
+    @Autowired
+    public HearingBookingDetailsController(MapperService mapperService) {
```

Comment:
I liked mrMapper ;)

Reply:
I thought id left it in there !!!

Reply:
mrMapper 🎉

Reply:
I concur that all occurrences of mapper service must now be called mrMapper :)